### PR TITLE
chore: Simplify NodeFeature to use FeaturesFeed

### PR DESCRIPTION
**Pull request by vinnyjth**
## DESCRIPTION

Rather than having two separate methods for rending features (the `NodeFeatures` way and the `FeedFeatures` way) I think it's worth unifying the approaches and having `NodeFeatures` render `FeedFeatures`. It was already 100% possible in the schema, just needed to be implemented on the front end. 

**Some caveats:** 

We are losing the `title` prop for `NodeFeatures`. I think that's okay, we were turning it off (passing a null string) in all of our client apps anyway.  

I added more fields back in to the skeleton feed features query, and included some comments. We need those fields because those features don't fetch their own fields. 

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/index.js [hunk 1]
```diff
@@ -65,7 +65,6 @@ export MediaControlsConnected, {
   MediaControls,
 } from './MediaControlsConnected';
 export NodeFeaturesConnected, {
-  NodeFeatures,
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/index.js:68]: we've made this mistake before, are we sure no one is using this?

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/index.js:68]: Screen Shot 2020-11-18 at 9 24 16 AM
Screen Shot 2020-11-18 at 9 24 02 AM
Screen Shot 2020-11-18 at 9 23 31 AM
Screen Shot 2020-11-18 at 9 23 17 AM
Screen Shot 2020-11-18 at 9 23 00 AM
Screen Shot 2020-11-18 at 9 22 17 AM

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/index.js:68]: @redreceipt I need to make a tool that does a search across our clients like this

**Review by redreceipt — APPROVED** [review]
